.circleci/config.yml

```diff
@@ -6,16 +6,13 @@ defaults: &defaults
 
   working_directory: ~/app
 
-
-
-
-
 jobs:
   commitlint:
     docker:
-      - image: williamlauze/circleci-commitlint:latest
+      - image: secretbase/circleci-commitlint:1.0.1
     working_directory: ~/app
-
+    environment:
+      CIRCLE_COMPARE_URL: << pipeline.project.git_url >>/compare/<< pipeline.git.base_revision >>..<<pipeline.git.revision>>
```

Comment:
Digging the source code at the upstream
https://github.com/wilau2/circleci-commitlint-step/blob/master/commitlint_range.sh#L28

`CIRCLE_COMPARE_URL` is not available in circle ci version 2.1 anymore
https://circleci.com/docs/2.0/env-vars/
https://discuss.circleci.com/t/is-circle-compare-url-still-a-thing/33366

So I just added an env var as example show at circle ci docs.
Hope this fix those noise when running commitlint ci job

https://circleci.com/docs/2.0/pipeline-variables/

https://github.com/wilau2/circleci-commitlint-step/issues/13